Gemfile

```diff
@@ -6,16 +6,25 @@ ruby File.read('.ruby-version').chomp
 # Reduces boot times through caching; required in config/boot.rb
 gem 'bootsnap', '>= 1.1.0', require: false
 
+gem 'activesupport', '~> 6.1.3'
```

Comment:
Why do you have to split these out?

Reply:
it was suggested at https://github.com/rails/rails/issues/41750#issuecomment-805913270 which removes the dependencies that depend on mimemagic, therefore allowing us to remove mimemagic

Reply:
it was suggested at https://github.com/rails/rails/issues/41750#issuecomment-805913270 which removes the dependencies that depend on mimemagic, therefore allowing us to remove mimemagic